bin/circle/merge-eyes-baselines

```diff
@@ -2,9 +2,12 @@
 
 require_relative '../../lib/cdo/git_utils'
 require_relative '../../lib/cdo/eyes_utils'
-
 # Temporarily disable all eyes branch merging
-exit(0)
+enabled = false
```

Comment:
is the original code technically 'enabled = true' in both of these examples? This PR appears to result in NOT running the original code in both cases.

Reply:
yeah, in the original code, this scripts were disabled.
The both had an "exit(0)" which made the rest of the code unreachable and therefor throwing that lint warning.
the intention is to still to NOT run this code. 

This is kind of a hacky way to disabling through code and avoid that warning.

The options were:
- Remove the code (but that would made us lose the ability to enable them easily
- Add a huge comment block. This throws a warning

Reply:
ah, got it!

Reply:
We do want to preserve existing behavior here. I agree that we don't want to either remove the code or comment it out, but I don't think we should change any functionality in pursuit of these fixes.

We've already got a comment and a puts statement explaining to developers why we are intentionally leaving this code inaccessible, I think we should just make sure we give that explanation in a way rubocop can also understand:

# rubocop:disable Lint/UnreachableCode

Reply:
See https://github.com/code-dot-org/code-dot-org/pull/47117 for an example (also, I wouldn't mind a review on that if you have a chance! :smile: )